code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -204,6 +204,26 @@
 		if("harm")
 			wuv(-1, M)
 
+/mob/living/simple_animal/pet/cat/attack_alien(mob/living/carbon/alien/humanoid/M)	//what the xeno was after all along in Alien
+	//. = ..()	//I assume this is needed

```

Comment:
Actually to draw attention to this because I forgot to remove the slashes during testing _if this code worked and not exhaustively for bugs_ and found no issue, I don't know if this is needed. I don't really know what it does or if it's needed but it was on another attacked by thing.